dolphin/api/v1/storages.py

```diff
@@ -83,7 +83,15 @@ def index(self, req):
         return storage_view.build_storages(storages)
 
     def show(self, req, id):
-        return dict(name="Storage 2")
+        try:
+            storage = db.storage_get(context, id)
+        except (AttributeError, exception.StorageNotFound, Exception) as e:
+            msg = _('Failed to get  Storage  from  DB: {0}'
+                    .format(e))
+            LOG.exception(msg)
+            raise exc.HTTPNotFound(explanation=msg)
```

Comment:
Here cannot raise HTTPNotFound exception except no item exist in database.

Reply:
Why need AttributeError, Exception here?
double spaces in msg, plz remove.

Reply:
done